<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Toni Gundogdu</author>
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>&amp;General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&amp;Path to c/clive:</string>
           </property>
           <property name="buddy">
            <cstring>ccliveEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ccliveEdit">
           <property name="toolTip">
            <string>Path to either cclive or clive command</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ccliveBrowse">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="abby.qrc">
             <normaloff>:/folder-saved-search.png</normaloff>:/folder-saved-search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="autodetectCcliveButton">
           <property name="toolTip">
            <string>Autodetect c/clive from the path</string>
           </property>
           <property name="text">
            <string>&amp;Autodetect</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="verifyCcliveButton">
           <property name="toolTip">
            <string>Verify path to c/clive</string>
           </property>
           <property name="text">
            <string>&amp;Verify</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Download videos to:</string>
           </property>
           <property name="buddy">
            <cstring>savedirEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="savedirEdit">
           <property name="toolTip">
            <string>Save downloaded videos to this directory</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="savedirButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="abby.qrc">
             <normaloff>:/folder-saved-search.png</normaloff>:/folder-saved-search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer_5">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Co&amp;nnection</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="proxyCombo">
           <property name="toolTip">
            <string>Whether a proxy should be used for the HTTP connections</string>
           </property>
           <item>
            <property name="text">
             <string>No Proxy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Use Proxy</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="proxyEdit">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="limitBox">
           <property name="toolTip">
            <string/>
           </property>
           <property name="text">
            <string>&amp;Limit transfer rate:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="limitSpin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>KB/s</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QCheckBox" name="timeoutBox">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Maximum time in seconds allowed the connection to the server to take</string>
           </property>
           <property name="text">
            <string>&amp;Timeout after:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="timeoutSpin">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="suffix">
            <string>s</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="socksBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Workaround the CURL SOCKS proxy bug by using
CURLOPT_TIMEOUT instead of CURLOPT_CONNECTTIMEOUT</string>
           </property>
           <property name="text">
            <string>&amp;Use SOCKS proxy workaround</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Misc</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="streamLabel">
           <property name="text">
            <string>S&amp;tream command:</string>
           </property>
           <property name="buddy">
            <cstring>streamEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="streamEdit">
           <property name="toolTip">
            <string>Stream command to be invoked when file transfer reaches
a certain percentage defined in the Options tab.

Example: /usr/local/bin/mplayer -really-quiet %i</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="streamButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="abby.qrc">
             <normaloff>:/folder-saved-search.png</normaloff>:/folder-saved-search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>E&amp;xternal command:</string>
           </property>
           <property name="buddy">
            <cstring>commandEdit</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="commandEdit">
           <property name="toolTip">
            <string>The command to be invoked when download finishes.
Any &quot;%i&quot; specifiers will be replaced by the name
of the downloaded video file.

Example: /usr/local/bin/mplayer %i</string>
           </property>
           <property name="whatsThis">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="commandButton">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="abby.qrc">
             <normaloff>:/folder-saved-search.png</normaloff>:/folder-saved-search.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="langLabel">
           <property name="text">
            <string>&amp;Language:</string>
           </property>
           <property name="buddy">
            <cstring>langCombo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="langCombo"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>99</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ccliveEdit</tabstop>
  <tabstop>ccliveBrowse</tabstop>
  <tabstop>autodetectCcliveButton</tabstop>
  <tabstop>verifyCcliveButton</tabstop>
  <tabstop>savedirEdit</tabstop>
  <tabstop>savedirButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>proxyCombo</tabstop>
  <tabstop>proxyEdit</tabstop>
  <tabstop>limitBox</tabstop>
  <tabstop>limitSpin</tabstop>
  <tabstop>timeoutBox</tabstop>
  <tabstop>timeoutSpin</tabstop>
  <tabstop>socksBox</tabstop>
  <tabstop>streamEdit</tabstop>
  <tabstop>streamButton</tabstop>
  <tabstop>commandEdit</tabstop>
  <tabstop>commandButton</tabstop>
  <tabstop>langCombo</tabstop>
 </tabstops>
 <resources>
  <include location="abby.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>proxyCombo</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onProxyChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>limitBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onLimitStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>PreferencesDialog</sender>
   <signal>finished(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onFinished(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>savedirButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onBrowseSaveDir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>98</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>timeoutBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onTimeoutStateChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ccliveBrowse</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onBrowseCclive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>streamButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onBrowseStreamCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onBrowseCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>116</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verifyCcliveButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onVerifyCclive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autodetectCcliveButton</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>onAutodetectCclive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onProxyChanged(int)</slot>
  <slot>onLimitStateChanged(int)</slot>
  <slot>onYoutubeToggled(bool)</slot>
  <slot>onFinished(int)</slot>
  <slot>onBrowseSaveDir()</slot>
  <slot>onBrowseCommand()</slot>
  <slot>onBrowseCclive()</slot>
  <slot>onLangChanged(int)</slot>
  <slot>onBrowseStreamCommand()</slot>
  <slot>onTimeoutStateChanged(int)</slot>
  <slot>onVerifyCclive()</slot>
  <slot>onAutodetectCclive()</slot>
 </slots>
</ui>
